internal/cloud/realm/dependencies.go

```diff
@@ -49,20 +82,43 @@ func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
 		return err
 	}
 
-	res, resErr := c.do(
-		http.MethodPost,
+	res, err := c.do(
+		http.MethodPut,
 		fmt.Sprintf(dependenciesPathPattern, groupID, appID),
 		api.RequestOptions{
 			Body:        body,
 			ContentType: w.FormDataContentType(),
 		},
 	)
-	if resErr != nil {
-		return resErr
+	if err != nil {
+		return err
 	}
 	if res.StatusCode != http.StatusNoContent {
-		return api.ErrUnexpectedStatusCode{"import dependencies", res.StatusCode}
+		return fmt.Errorf("import got invalid status code %d", res.StatusCode)
+	}
+
+	waitForTranspilation := func() error {
```

Comment:
I used a function for this to be consistent with `deployDraftAndWait` but could be removed into a basic loop in same scope.

Reply:
Yes I'll change all the mentions of transpilation. Moving out of the `ImportDependencies` is what I meant above when talking about `importDependenciesAndWait`.